charts/latest/spdk-csi/values.yaml

```diff
@@ -51,21 +59,11 @@ externallyManagedSecret:
   # Specifies whether a externallyManagedSecret should be created
   create: true
 
-spdkdev:
-  # Specifies whether a spdkdev should be created
-  create: false
-
-storageclass:
-  create: true
-
-controller:
-  replicas: 1
-
 # Configuration for the CSI to connect to the cluster
 csiConfig:
   simplybk:
     uuid: 963c9d0a-4506-43c3-a722-0b7c8b157038
-    ip: 13.58.2.30
+    ip: https://o5ls1ykzbb.execute-api.eu-central-1.amazonaws.com
```

Comment:
This will be a default value? Can we change these?

Reply:
Yes, this can be changed through Helm parameters